test/metric_value_benchmark/agent_configs/swap_config.json

```diff
@@ -0,0 +1,23 @@
+{
+    "agent": {
+        "metrics_collection_interval": 60,
+        "run_as_user": "root",
+        "debug": true,
+        "logfile": ""
+    },
+    "metrics": {
+        "namespace": "MetricValueBenchmarkTest",
+        "append_dimensions": {
+            "InstanceId": "${aws:InstanceId}"
+        },
+        "metrics_collected": {
+            "swap": {
+                "measurement": [
+                    "free",
+                    "used",
+                    "used_percent"
+                ]
```

Comment:
**Nitpick**: Maybe add some default features such as `metrics_collection_interval` or `append_dimensions` so we can increase confidence between backward compatibility and also cover more features

Reply:
Another point of me would be would we reduce the `metrics_collection_interval` to reduce the amount of time the test needs to be run.

Reply:
nice ideas! i'll add them

Reply:
Those can be future improvement too so ! Its fine to not doing it now and create ticket for it.

Reply:
How do you intend to test metric collection interval if you add it?